content/root/platform-customization.textile

```diff
@@ -40,7 +39,7 @@ As part of our Enterprise package offering, Ably are able to offer customers ded
 
 This involves the provision of dedicated realtime and routing servers, with shared database cluster and common services (such as stats, logging, registries, and configuration).
 
-We provide you with custom endpoints for your cluster, for example: @yourcompany-realtime.ably.io@ and @yourcompany-realtime.ably.io@.  Additionally, we can support CNAMEs using your own custom root domain.
+We provide you with custom endpoints for your cluster, for example: @yourcompany.realtime.ably.net@ and @yourcompany.rest.ably.net@.  Additionally, we can support CNAMEs using your own custom root domain.
```

Comment:
The switch to the `endpoint` option means that we do not have separate domains for `realtime` and `rest` endpoints, they use the same domain.

```suggestion
We provide you with a custom endpoint for your cluster, for example: @yourcompany.realtime.ably.net@.  Additionally, we can support CNAMEs using your own custom root domain.
```